client/foundation/qml/Settings/SettingsComponents/Profile.qml

```diff
@@ -72,11 +72,8 @@ ColumnLayout {
                     width: cornerCarat.width * 0.2
                     cursorShape: Qt.PointingHandCursor
                     onClicked: {
-                        print(Herald.config.profilePicture)
-                        const imageDimensions = JSON.parse(
-                                                  Herald.utils.imageDimensions(
-                                                      "file://" + Herald.config.profilePicture))
-                        print("opening the crop window not yet implemented")
+
+                        //  cropCallbackArg(Herald.config.profilePicture)
```

Comment:
This is the stub which should open the cropper on a preexisting profile picture. it is not working on an absolute path. likely the solution is to append the directory of the executable to the file url.